### PR TITLE
Allow ternary operator

### PR DESCRIPTION
**Pull request by wowawiwa**

### CODE REVIEW

**Comment by psychocandy** [comment]
I think we should allow these if it's short and not too complex. i.e less than x amount of chars. wdys?

**Comment by miry** [comment]
given: 
```ruby
a = 1
b = 2 
c = 3
```
Example of small use:
```ruby
x = a > 1 ? b : c
```

Example of logical:
```ruby
x = a > 1 && b || c
```

Example of `If statements`:
```ruby
x = c
x = b if a > 1
```

Example of `multi line If statements`:
```ruby
x = if a > 1
        b
      else
        c
      end
```

For this is good to use ternary. But in Rails we usually have AR objects and variables.

File: style/ruby/README.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ Ruby
 * Avoid conditional modifiers (lines that end with conditionals).
 * Avoid multiple assignments per line (`one, two = 1, 2`).
 * Avoid organizational comments (`# Validations`).
-* Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
   instead to emphasize code branches.
```
Comment (miry) [inline style/ruby/README.md]: I suppose you need to remove this line also.

**Comment by miry** [comment]
another case why multi line if is good, when you start debug and use step by step.

**Pull request closed without merging**